ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -21,6 +21,16 @@ sriov "FPGA Programmable Acceleration Card N3000 for Networking"
 # TESTNAME
 dpdk "Client should be able to forward packets"
 
+# SKIPTEST
+# bz### https://issues.redhat.com/browse/OCPBUGS-11046
+# TESTNAME
+tuningcni "sysctl allowlist update should start a pod with custom sysctl only after adding sysctl to allowlist"
```

Comment:
This is duplicated in 4.13 skiplist. Which versions do you want to skip it? This section in particular will skip in ALL versions.

Reply:
Yes, I actually wanted to skip for 13 and 14... there are 2 different bugs 1 for 13 and 1 for 14, so that's why I defined it that way. What do you suggest?

Reply:
Add it to `function create_tests_temp_skip_list_13` and `function create_tests_temp_skip_list_14`